pages/workspace/create/tdei.vue

```diff
@@ -129,42 +139,50 @@ const importer = new TdeiImporter(workspacesClient, tdeiClient, osmClient, conte
 
 const loading = reactive(new LoadingContext());
 const route = useRoute();
-const tdeiRecordId = route.query.tdeiRecordId;
-const datasetRequested = tdeiRecordId?.length ?? false;
+const tdeiRecordId = ref(null);
 const record = reactive({});
 const map = ref({});
 const workspaceTitle = ref('');
 const projectGroupId = ref(null);
 
+watch(tdeiRecordId, (val) => getDatasetInfo(val));
+
 const complete = computed(() =>
   workspaceTitle.value.trim().length > 0
     && projectGroupId.value !== null
-    && tdeiRecordId !== undefined
+    && tdeiRecordId !== null
 );
 
-async function getDatasetInfo(route, params) {
-  await loading.wrap(tdeiClient, async (client) => {
-    const info = await client.getDatasetInfo(tdeiRecordId);
-
-    if (!info) {
-      return;
+async function getDatasetInfo(id: string) {
+  if (id === null) {
+    for (const prop in record) {
+      record[prop] = '';
     }
 
+    workspaceTitle.value = '';
+    return;
+  }
+
+  await loading.wrap(tdeiClient, async (client) => {
+    const info = await client.getDatasetInfo(id);
+
     for (const prop in info) {
       record[prop] = info[prop];
     }
 
-    workspaceTitle.value = record.metadata?.dataset_detail?.name ?? '';
-    projectGroupId.value = record.project_group.tdei_project_group_id;
-  })
+  });
+
+  await nextTick();
+
+  workspaceTitle.value = record.metadata?.dataset_detail?.name ?? '';
+  projectGroupId.value = record.project_group.tdei_project_group_id;
+  tdeiRecordId.value = record.tdei_dataset_id;
```

Comment:
Any reason to reassign this? It looks like these two values are always equal at this point.

```suggestion
```